libs/wingsdk/src/target-tf-azure/bucket.inflight.ts

```diff
@@ -57,10 +58,18 @@ export class BucketClient implements IBucketClient {
    * @param key Key of the object
    * @param body string contents of the object
    */
-  public async put(key: string, body: string): Promise<void> {
-    await this.containerClient
-      .getBlockBlobClient(key)
-      .upload(body, body.length);
+  public async put(
+    key: string,
+    body: string,
+    props?: BucketPutProps
+  ): Promise<void> {
+    const blobClient = this.containerClient.getBlockBlobClient(key);
+    const options = {
+      blobHTTPHeaders: {
+        blobContentType: props?.contentType,
```

Comment:
Missing auto detect or does this happen automatically in azure? If so, better document.